src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/remoteJob/BuildInfoExporterAction.java

```diff
@@ -94,12 +128,20 @@ public void buildEnvVars(AbstractBuild<?, ?> build, EnvVars env) {
                 }
             }
             if (lastBuild != null) {
-                env.put(BUILD_NUMBER_VARIABLE_PREFIX + sanatizedBuildName, Integer.toString(lastBuild.buildNumber));
-                env.put(BUILD_RESULT_VARIABLE_PREFIX + sanatizedBuildName, lastBuild.buildResult.toString());
+                env.put(JOB_NAME_VARIABLE, lastBuild.projectName);
+                env.put(BUILD_NUMBER_VARIABLE_PREFIX + sanatizedProjectName, Integer.toString(lastBuild.buildNumber));
+                env.put(BUILD_URL_VARIABLE_PREFIX + sanatizedProjectName, lastBuild.jobURL.toString());
+                env.put(BUILD_RESULT_VARIABLE_PREFIX + sanatizedProjectName, lastBuild.buildResult.toString());
             }
         }
     }
 
+    public static String sanitizeProjectName(String project)
```

Comment:
Not sure this sanitization is correct.
Jenkins core has different pattern fo sure